spotipy/json_types.py

```diff
@@ -0,0 +1,573 @@
+import sys
+from typing import Any, Generic, List, Mapping, TypeVar, overload
+
+if sys.version_info >= (3, 8):
+    from typing import Literal, TypedDict
+else:
+    from typing_extensions import Literal, TypedDict
+
+_T = TypeVar("_T", covariant=True)
+
+
+TokenInfo = TypedDict("Token", {
```

Comment:
I would suggest changing `"Token"` to `"TokenInfo"` to match the name of the variable.